packages/utils/node-resolver-core/src/NodeResolver.js

```diff
@@ -313,7 +313,7 @@ export default class NodeResolver {
             'https://parceljs.org/features/node-emulation/#polyfilling-%26-excluding-builtin-node-modules',
         });
 
-        await packageManager.resolve(packageName, this.projectRoot + '/index', {
+        await packageManager.resolve(builtin, this.projectRoot + '/index', {
           saveDev: true,
```

Comment:
By the way, should not this be a non-dev dependency?

Reply:
Maybe? Two reasons I made it a dev dep:

1. It was a dev dependency before when it was included in Parcel itself.
2. The "process" dependency might actually be used not in your project, but by a dependency in node_modules. We cannot add the dep to that, so we add it to the project root instead. So it's kinda an indirect dependency, needed by Parcel not by your project.

I could be convinced otherwise if you feel strongly.